lib/test_suite.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, suite_path, args):
                 dict.fromkeys(self.ini[i].split())
                 if i in self.ini else dict())
 
-        self.parse_bool_opt('pre_cleanup', False)
+        self.parse_bool_opt('pre_cleanup', True)
```

Comment:
To be honest, I don't sure we should enable the option by default in test-run. Say, ok, we decided to use this way in tarantool testing. But test-run is the common harness for testing tarantool-oriented project and I tentative about pushing users to understand and use this extra logic.